### ISSUE TITLE
Rating with custom images don't work as expected. They require slide-gesture instead of tap.

### ISSUE THREAD

**Issue by tan116** [Monte9/react-native-ratings#19]

**Comment by KishanGujarati** [comment]
@tan116 
you can use this

`const img = require('./rating.png')`

`<Rating
  type='custom'
  ratingImage={img}
  ratingCount={5}
  imageSize={30}
  onFinishRating={this.ratingCompleted}
  style={{ paddingVertical: 10 }}
/>`

i hope it will help you....

**Comment by tan116** [comment]
@KishanGujarati  I don't see what you've done differently. If you're referring to the "const img = ..." line, I've obviously done that.
In simpler words, the issue is this:
Tapping on a star doesn't work because the rating returned is always constant.
Sliding on the rating increases/decreases the rating _but that behavior is extremely non-intuitive_.

**Comment by KishanGujarati** [comment]
@tan116 
you should try this
`<AirbnbRating
                    count={5}
                    defaultRating={3}
                    size={25}
                    onFinishRating={this.ratingCompleted}
                  />`

Tapping behavior working in this...but actually problem is that only tapping behavior is working in this , sliding behavior not working in this.....

**Comment by Monte9** [comment]
Yes, `<AirbnbRating />` provides tapping behavior with simple animations. 

I'll try to fix this issue in the next release soon.

**Comment by tan116** [comment]
Thanks. Looking forward to it

**Comment by michaelVictoriaDev** [comment]
Any update on this ? :)
